minidump-processor/src/symbols.rs

```diff
@@ -159,6 +159,7 @@ pub trait SymbolProvider {
     /// Keys are implementation dependent.
     /// For example the file name of the module (code_file's file name).
     fn stats(&self) -> HashMap<String, SymbolStats>;
+    fn pending_stats(&self) -> PendingSymbolStats;
```

Comment:
Can you document this please so implementors know what they have to do?

I tried making these things have a default impl, which might even work for non-async functions. Though the compiler complained to me about Object Safety when I tried to add a default impl for `get_file_path`.

Reply:
done